e2e_samples/parking_sensors/src/ddo_transform/Dockerfile

```
@@ -1,9 +1,14 @@
-FROM python:3.7.3
+FROM python:3.11
```

Same as previous comment: 
```suggestion
FROM --platform=linux/amd64 python:3.11
```